config/initializers/secure_headers.rb

```diff
@@ -1,12 +1,15 @@
 ::SecureHeaders::Configuration.default do |config|
-  config.hsts = false
+  config.hsts = "max-age=#{20.years.to_i}; includeSubdomains"
```

Comment:
Are you sure this is the same as above?

Reply:
@duduribeiro what about this?

Reply:
this is the same behave as before:

![image](https://user-images.githubusercontent.com/771411/75452056-536b0600-5950-11ea-8a7e-fd08a9864c43.png)

I removed the includeSubdomains now